src/core_plugins/kibana/public/discover/components/field_chooser/field_chooser.html

```diff
@@ -54,16 +54,16 @@ <h3 class="sidebar-list-header-label" id="available_fields" tabindex="0">Availab
         ng-class="{ 'kuiButton--basic': !filter.active, 'kuiButton--primary': filter.active, 'hidden-xs': !showFields, 'hidden-sm': !showFields }"
         class="kuiButton kuiButton--small pull-right discover-field-filter-toggle"
         ng-click="showFilter = !showFilter"
-        aria-label="Show field settings"
-        aria-haspopup="true"
-        aria-expanded="{{showFilter}}"
+        aria-label="{{showFilter ? 'Hide' : 'Show'}} field settings"
+        aria-expanded="{{!!showFilter}}"
+        aria-controls="discover-field-filter"
       >
         <span aria-hidden="true" class="kuiIcon fa-gear"></span>
       </button>
     </h3>
   </div>
 
-  <div class="sidebar-item discover-field-details" ng-show="showFilter">
+  <div class="sidebar-item discover-field-details" ng-show="showFilter" id="discover-field-filter">
```

Comment:
Really minor nit, but per our [HTML style guide](https://github.com/elastic/kibana/blob/master/style_guides/html_style_guide.md#camel-case-id-and-other-attribute-values), can we write IDs in camel case, for both this ID and the other ones we're adding?